intro-to-R-tidyverse/01-intro_to_base_R.Rmd

```diff
@@ -154,7 +154,7 @@ We can use pre-built computation methods called "functions" for other operations
 Functions have the following format, where the _argument_ is the information we are providing to the function for it to run. 
 An example of this was the `atan()` function used above.
 
-```{r, eval=FALSE}
+```r
```

Comment:
Moving my comment here, as this is file that is actively modified...

This is not a real function, so it will generate a code if anybody tries to run it. So I made it a fenced code block, not a "chunk". The only concern here is that it will look different in the RStudio interface (won't get the chunk background). I could go either way.